src/main/java/mods/railcraft/common/core/Railcraft.java

```diff
@@ -210,7 +213,7 @@ public void serverCleanUp(FMLServerStoppingEvent event) {
     public void missingMapping(FMLMissingMappingsEvent event) {
         for (FMLMissingMappingsEvent.MissingMapping mapping : event.get()) {
             if (mapping.type == GameRegistry.Type.BLOCK) {
-                Block block = GameRegistry.findBlock(MOD_ID, MiscTools.cleanTag(mapping.name));
+                Block block = GameRegistry.findBlock(MOD_ID, mapping.resourceLocation.getResourcePath());
```

Comment:
Just in case, to save you debugging headaches, this event is currently not working: https://github.com/MinecraftForge/MinecraftForge/issues/2491